packages/stores/src/account/base.ts

```diff
@@ -126,6 +131,20 @@ export class AccountStore<Injects extends Record<string, any>[] = []> {
 
   private _cache = new LRUCache<string, CacheEntry>({ max: 30 });
 
+  /**
+   * We make sure that the 'base' field always has as its value the native chain parameter
+   * and not values derived from the IBC connection with Osmosis
+   */
+  private get _assets() {
```

Comment:
What are your thoughts on renaming this to `walletManagerAssets` since we only need this for cosmos-kit?

Reply:
Yes makes sense

Reply:
Done